api/ttn/lorawan/v3/user.proto

```diff
@@ -43,6 +43,13 @@ enum ConsoleTheme {
   CONSOLE_THEME_DARK = 2;
 }
 
+// Tutorial is a set of possible tutorials in the Console.
+enum Tutorial {
+  option (thethings.json.enum) = {marshal_as_string: true};
+
+  TUTORIAL_LIVE_DATA_SPLIT_VIEW = 0;
```

Comment:
Add a zero value for unknown/unspecified. Otherwise we can't differentiate the zero value from a particular value. This is a best practice for (proto) enums.